vignettes/iSEE_vignette.Rmd

```diff
@@ -135,5 +134,15 @@ file.copy(from = temp_file, to = newLocation)
 
 # FAQ
 
-*Coming soon.*
+**Q: Can you implement a 'Copy to clipboard' button in the code editor?**
```

Comment:
😄

Reply:
let's see how many users don't read the vignette ;)